src/Network/Network/AzureFirewall/NetworkRule/NewAzureFirewallNetworkRuleCommand.cs

```diff
@@ -22,9 +22,11 @@
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FirewallNetworkRule", SupportsShouldProcess = true), OutputType(typeof(PSAzureFirewallNetworkRule))]
-    public class NewAzureFirewallNetworkRuleCommand : NetworkBaseCmdlet
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FirewallNetworkRule", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet), OutputType(typeof(PSAzureFirewallNetworkRule))]
+    public class NewAzureFirewallNetworkRuleCommand : AzureFirewallBaseCmdlet
     {
+        private const string DefaultParameterSet = "Default";
```

Comment:
You can use the string defined in `AzureFirewallApplicationRuleParameterSets`.

Reply:
Removed